### ISSUE TITLE
Write store category copy

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#636]
Write descriptions for each of [the store collections](https://store.cncf.io/collections). The description will fit in here:
<img width="897" alt="Screen Shot 2022-12-08 at 10 48 39 AM" src="https://user-images.githubusercontent.com/463160/206541531-0ae02208-1ca9-429f-967d-8a675f23d508.png">

**Comment by bhavyastar** [comment]
Hey @cjyabraham , Would you please assign this to me?

**Comment by KTan1226** [comment]
Hi @cjyabraham - is the Shopify contractor moving this forward?

**Comment by cjyabraham** [comment]
No, that's not their skillset. We were hoping Charley or someone on your team could do it.
